.github/workflows/linters.yml

```diff
@@ -0,0 +1,3151 @@
+
```

Comment:
Kindly fix your linters file, follow the instructions [here](https://github.com/microverseinc/linters-config/tree/master/html-css) and replace your linters.yml file with this [one](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml). Linters are beneficial for code the standardization. 🟢